Lib/unittest/case.py

```diff
@@ -331,8 +333,8 @@ class TestCase(object):
         the instance's assertion methods fail; test methods raising this
         exception will be deemed to have 'failed' rather than 'errored'.
     * longMessage: determines whether long messages (including repr of
-        objects used in assert methods) will be printed on failure in *addition*
-        to any explicit message passed.
+        objects used in assert methods) will be printed on failure in
+        *addition* to any explicit message passed.
```

Comment:
To conform with [PEP 8 E501](https://www.flake8rules.com/rules/E501.html).

> Line lengths are recommended to be no greater than 79 characters. The reasoning for this comes from PEP8 itself:
> 

`Limiting the required editor window width makes it possible to have several files open side-by-side, and works well when using code review tools that present the two versions in adjacent columns.`

> 
> It is fairly common for developers, especially those in closed-source projects, to change the maximum line length to 100 or 120 characters.